### PR TITLE
Pimped up the 3D model of AG12864E

### PR DESCRIPTION
**Pull request by Misca1234**
This is the source code for the AG12864E 3D model

The footprint is here
https://github.com/KiCad/Displays.pretty/pull/10

bild

kicadlibrary push here
https://github.com/KiCad/kicad-library/pull/1367

package3D push here
https://github.com/KiCad/packages3D/pull/71

### CODE REVIEW

**Comment by poeschlr** [comment]
The credits file should go into the packages3d repo. (sorry for the confusion.)

File: Displays.3dshapes/CREDITS.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+### Info requirements:
+- Author
+- if the 3D model is script generated, a link to the script generator is required as well as the version and sw used
+- if the 3D model is manually generated, include the source model @ https://github.com/KiCad/packages3D-source
+
+<hr>
+
+### AG12864E.FCStd
+Author:	Stefan Olsson
+Tool:	- FreeCAD 0.17 Build 11223 64 bit Microsoft windwos platform https://www.freecadweb.org/
```
Comment (pointhi) [inline Displays.3dshapes/CREDITS.md]: there is a little spelling error in ```windows```

**Comment by Misca1234** [comment]
huh,  have you never heard of windwos, It is their Linux distribution ;)
I fix it

**Comment by Misca1234** [comment]
Fixed

**Comment by poeschlr** [comment]
credits file in the wrong repo.

**Comment by Misca1234** [comment]
Removed

**Comment by poeschlr** [comment]
Thanks for your contribution

**Comment by Misca1234** [comment]
Thanks

**Pull request merged**